docs/custom-extensions/busola-web-components.md

```diff
@@ -0,0 +1,117 @@
+| **placeholder**                   | No       | string   | Specifies a short hint about the input field value.                                                                                                                                       |
+| **language**                      | No       | string   | Specifies the programming language of the editor's content (e.g., `javascript`, `json`). Defaults to `javascript`.                                                                        |
+| **height**                        | No       | string   | Specifies the height of the component. Must include the unit (e.g., `100px`, `50vh`).                                                                                                     |
+| **schema-id**                     | No       | string   | A unique identifier for the JSON schema used to enable autocompletion and validation. If not provided, autocompletion is disabled.                                                        |
+| **autocompletion-disabled**       | No       | boolean  | Disables autocompletion suggestions when set to `true`.                                                                                                                                   |
+| **read-only**                     | No       | boolean  | Specifies if the field is read-only. Defaults to `false`.                                                                                                                                 |
+| **on-change**                     | No       | function | Callback function triggered when the content changes.                                                                                                                                     |
+| **on-mount**                      | No       | function | Callback function triggered when the editor mounts.                                                                                                                                       |
+| **on-blur**                       | No       | function | Callback function triggered when the editor loses focus.                                                                                                                                  |
+| **on-focus**                      | No       | function | Callback function triggered when the editor gains focus.                                                                                                                                  |
+| **update-value-on-parent-change** | No       | boolean  | Updates the editor content if the parent component changes its `value` prop.                                                                                                              |
+| **options**                       | No       | object   | Custom options for configuring the Monaco Editor. Refer to the Monaco Editor API for available options.                                                                                   |
```

Comment:
Can we add a link to the Monaco Editor API docs? Is it this one - https://microsoft.github.io/monaco-editor/docs.html ?